### ISSUE TITLE
Textures without a `source` field crash the importer

### ISSUE THREAD

**Issue by MeFisto94** [KhronosGroup/glTF-Blender-IO#1395]
**Describe the bug**
When importing a gltf file that contains no "regular" `source` field (i.e. it's only available as part of a (required) extension, with no fallback provided), the importer crashes here:

https://github.com/KhronosGroup/glTF-Blender-IO/blob/b44e08ec605d9602567cc6becb338277f66dbb80/addons/io_scene_gltf2/io/com/gltf2_io.py#L1120

With callstacks further down from https://github.com/KhronosGroup/glTF-Blender-IO/blob/b44e08ec605d9602567cc6becb338277f66dbb80/addons/io_scene_gltf2/io/com/gltf2_io.py#L1181 etc

**To Reproduce**
Craft a GLTF file only containing DDS Format Textures (I'll comment on the other relevant issues about their support).
I am not completely sure if that is okay according to the spec, but I think it should be, if no loader for that extension is to be found, then the texture just cannot be displayed/the importer should fail safe.

**Expected behavior**
Loading the texture and then failing at the later step of "the extension is not supported at this addon version", much like it does when I just add a dummy `source: i` field, that mirrors the same as the extension content.

**.blend file/ .gltf**
I can't include the files in question, but a simple:
```json
"textures": [
    {
        "extensions": {
            "MSFT_texture_dds": {
                "source": 0
            }
        }
    }
],
```
together with adding that string to both `extensionsUsed` and `extensionsRequired` should do.

**Version**
 - OS: Windows
 - Blender Version 2.83

**Comment by scurest** [comment]
This was fixed by #1187, which went into Blender 2.91. You just need to upgrade your Blender version.

**Comment by MeFisto94** [comment]
Apologizes for not searching for that more thoroughly and thanks for the fix
